course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -43,6 +43,10 @@
                 $scope.lookup(searchTerm);
             }
         };
+        $scope.roleselected = function(role_id, role_name){
```

Comment:
Heh, I meant the params as well as the method name.  Don't sweat it, I'll tweak it while I'm doing the user lookup.